src/components/Signup/Signup.js

```diff
@@ -81,15 +81,36 @@ export default injectIntl(function({ intl }) {
       <article className="main-panel">
         <header>
           <h1 className="logo-doppler-new">Doppler</h1>
-          <small className="content-signin">
-            {_('signup.do_you_already_have_an_account')}{' '}
-            <Link to="/login" className="link-green uppercase">
-              {_('signup.log_in')}
-            </Link>
-          </small>
+          <div className="language-selector">
+            {intl.locale === 'es' ? (
```

Comment:
Did you test if it is work fine after a change?

Reply:
I said that because `intl.locale` is not part of props or state, right?

Reply:
yes it's works. in the function has the dependency